asdf/tags/wcs/wcs.py

```diff
@@ -295,3 +314,47 @@ def assert_equal(cls, old, new):
         assert old.name == new.name
         for old_frame, new_frame in zip(old.frames, new.frames):
             helpers.assert_tree_match(old_frame, new_frame)
+
```

Comment:
Based on an email from @nden it sounds like we need to have a discussion about coordinate serialization in general in ASDF.